<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Storage Tester - Scan Files &amp; Storage Update Times</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,1,0">
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Running Time</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblRunningTime">
        <property name="text">
         <string>#</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="btnStartStop">
          <property name="styleSheet">
           <string notr="true">QPushButton {
	padding-top: 10px;
    padding-bottom: 10px;
}</string>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
             <item>
              <widget class="QGroupBox" name="boxScanDir">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="title">
                <string>Scan Type</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QCheckBox" name="chkScanFiles">
                  <property name="font">
                   <font>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Generic Scan (creates files in %TEMP%/StorageTester)</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="Line" name="line">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="layoutDirectorySelector" native="true">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="font">
                        <font>
                         <weight>50</weight>
                         <bold>false</bold>
                        </font>
                       </property>
                       <property name="text">
                        <string>Select Directory</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="btnSelectDirectory">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="font">
                        <font>
                         <weight>50</weight>
                         <bold>false</bold>
                        </font>
                       </property>
                       <property name="text">
                        <string>Select Directory</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="font">
                      <font>
                       <weight>75</weight>
                       <bold>true</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string>Directory:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="lblSelectedDirecvtory">
                     <property name="font">
                      <font>
                       <weight>50</weight>
                       <italic>true</italic>
                       <bold>false</bold>
                      </font>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="wordWrap">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="boxGenericScan">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="title">
                <string>Generic Scan using %TEMP% directory for work</string>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QCheckBox" name="chkRefresh">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="font">
                   <font>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Refresh Storage Space Left (after each file scan)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="chkCalcCRC">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="font">
                   <font>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Calculate CRC of scanned file</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="chkEnableDirectoryCreation">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="font">
                   <font>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Create directories (for Scan Files option)</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QWidget" name="layoutFileNumberOptions_2" native="true">
             <layout class="QVBoxLayout" name="layoutFileNumberOptions">
              <item>
               <widget class="QCheckBox" name="chk50">
                <property name="text">
                 <string>50</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk100">
                <property name="text">
                 <string>100</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk200">
                <property name="text">
                 <string>200</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk500">
                <property name="text">
                 <string>500</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk1000">
                <property name="text">
                 <string>1000</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk2000">
                <property name="text">
                 <string>2000</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk5000">
                <property name="text">
                 <string>5000</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="chk10000">
                <property name="text">
                 <string>10000</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QTextEdit" name="output"/>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,1">
      <item>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>File Scan Information</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblNumFilesScanned">
          <property name="text">
           <string># Files Scanned:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblAverageScanTime">
          <property name="text">
           <string>Average Scan Time</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Storage Scan Information</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblDrive">
          <property name="text">
           <string>Drive:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblAverageStorageRefreshTime">
          <property name="text">
           <string>Average Storage Refresh Time</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="chkVerboseLogging">
      <property name="text">
       <string>Verbose Logging</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
